src/ServiceContainer/SymfonyExtension.php

```diff
@@ -75,6 +77,10 @@ public function load(ContainerBuilder $container, array $config): void
 
         $this->loadBootstrap($this->autodiscoverBootstrap($config['bootstrap']));
 
+        if (empty($config['bootstrap'])) {
```

Comment:
If using an autodiscovered bootstrap file, it would load the env variables two times, we should check for that.

Additionally, we should mention that in docs - might be not that obvious that it only happens if there's no explicitly passed bootstrap file.